components/two_thingy/twowrap.js

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import Two from "two.js";
+import { useEffect } from "react";
+
+export default function TwoWrapper() {
+  let two = new Two({
+    autostart: true,
+    fullscreen: false,
+  });
+  let twoRef = React.createRef();
```

Comment:
Either import everything from React (*) or limit to the needed elements. I suggest the second for performance.